RecoPixelVertexing/PixelLowPtUtilities/plugins/PixelVertexProducerClusters.h

```diff
@@ -1,11 +1,13 @@
 #ifndef PixelVertexProducerClusters_H
 #define PixelVertexProducerClusters_H
 
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
 #include "FWCore/Framework/interface/global/EDProducer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
-
 #include "DataFormats/TrackerRecHit2D/interface/SiPixelRecHitCollection.h"
 
+class TrackerGeometry;
```

Comment:
I think we don't allow forward declaring classes from other packages [1]. Was there a reason to not include the header?

[1] (https://cms-sw.github.io/cms_coding_rules.html 4.7)

Reply:
done, though there are other examples of that in this package:

https://github.com/cms-sw/cmssw/blob/dfffa4a11f400b84082bfd53a6de14ced9800d59/RecoPixelVertexing/PixelLowPtUtilities/plugins/PixelVertexProducerClusters.h#L15